lib/slack/handlers.ex

```diff
@@ -72,6 +72,10 @@ defmodule Slack.Handlers do
     end
   end)
 
+  def handle_slack(%{type: "im_created", channel: channel}, slack) do
```

Comment:
Yeah, I guess we can just to keep the internal data structure up-to-date.  Honestly though, I can't imagine how that information is useful to a bot.  It merely indicates that a human has opened or closed the channel in their client, which has no bearing on a bot's ability to send messages.  The channel cannot be "closed," that I can see.